### DISCUSSION TITLE
Allow non-collaborators to vote to reopen closed Issues

### DISCUSSION THREAD

**Discussion by DillPickle80** [community/community#14061] [Projects and Issues]
Why is it that almost every issues i find while searching for an active problem ends up being closed before it seems to be actually solved? This is so irritating, there should be a feature for non-collaborators to ask to reopen them, i've seen people start new issues just to then be closed because it's a duplicate... of an issue that is closed but is still an issue, otherwise it wouldn't get posted again and i wouldn't be searching for those... At least, i don't know, leave some way to contact actual collaborators or something, because right now if the collaborators don't display their email there's just no way. Once i had to find a "hack" to actually retrieve the private email of a collaborator just to tell him the damn issue wasn't solved even if closed... 

Sorry if i sound salty, but it piss me off a little.

**Comment by Marcono1234** [comment]
Related to #5084

**Reply by gerib** [reply]
This is a different story

> Let issue authors reopen their own issues

than here.

**Reply by Marcono1234** [reply]
Yes they are definitely not the same, and I am not saying that either of them should be closed as duplicate. But the issue that maintainers close an issue and are unaware that other users cannot reopen it or don't watch the activity on the issue are similar. In both cases it results in confusion, possibly frustration and duplicate issues.

**Comment by gerib** [comment]
I second this feature request. Reason: https://github.com/eclipse-m2e/m2e-core/issues/493#issuecomment-2563174486.

**Comment by lizi83412** [comment]
Allowing non-collaborators to vote to reopen closed issues can improve transparency and user involvement, especially for platforms like a family-friendly entertainment app with parental controls designed [here](https://castleapp.co/) to provide safe and engaging content for children while giving parents the tools to monitor and manage their child's digital experience

**Comment by marcindulak** [comment]
Collected a few more discussions asking for that feature in https://github.com/orgs/community/discussions/182939.
Consider going to all of them an upvoting them.
